katdal/h5datav3.py

```diff
@@ -45,11 +45,11 @@
 SENSOR_PROPS.update({
     '*activity': {'greedy_values': ('slew', 'stop'), 'initial_value': 'slew',
                   'transform': lambda act: SIMPLIFY_STATE.get(act, 'stop')},
-    '*target': {'initial_value': '', 'transform': _robust_target},
     '*ap_indexer_position': {'initial_value': ''},
-    '*_serial_number': {'initial_value': 0},
-    '*dig_noise_diode': {'categorical': True, 'greedy_values': (True,),
-                         'initial_value': 0.0, 'transform': lambda x: x > 0.0},
+    '*noise_diode': {'categorical': True, 'greedy_values': (True,),
+                     'initial_value': 0.0, 'transform': lambda x: x > 0.0},
+    '*serial_number': {'initial_value': 0},
```

Comment:
You dropped an underscore here was it intended ?

Reply:
Yes. I also sorted them alphabetically as a clean-up. It just reads easier without changing the behaviour.